release.sh

```diff
@@ -6,6 +6,12 @@ VERSION_NO_V=$(echo $1 | tr -d "\nv")
 VERSION="v"$VERSION_NO_V
 DATE=$(date "+%Y-%m-%d")
 
+echo "Sanity checks"
+
+read -p "Are you on the master branch?" -n 1 -r
+read -p "Have you amended CHANGELOG.md and git push it?" -n 1 -r
```

Comment:
let's make use a git command to make sure there are no changes in the working tree or index (i.e. nothing uncommitted)